src/vs/editor/contrib/codeAction/browser/codeActionMenu.ts

```diff
@@ -203,6 +204,21 @@ export class CodeActionMenu extends Disposable implements IEditorContribution {
 		const renderDisposables = new DisposableStore();
 		const renderMenu = document.createElement('div');
 
+		// Render invisible div to block mouse interaction in the rest of the UI
+		const menuBlock = document.createElement('div');
+		this.block = element.appendChild(menuBlock);
+		this.block.classList.add('context-view-block');
+		this.block.style.position = 'fixed';
+		this.block.style.cursor = 'initial';
+		this.block.style.left = '0';
+		this.block.style.top = '0';
+		this.block.style.width = '100%';
+		this.block.style.height = '100%';
+		this.block.style.zIndex = '-1';
+
+		// TODO@Steven: this is never getting disposed
+		dom.addDisposableListener(this.block, dom.EventType.MOUSE_DOWN, e => e.stopPropagation());
```

Comment:
You likely need to do something with the disposable that `dom.addDisposableListener` returns, such as adding it to the `renderDisposables`